app/src/androidTest/java/com/github/epfl/meili/poi/PoiActivityTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.github.epfl.meili.poi
+
+
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@LargeTest
+@RunWith(AndroidJUnit4::class)
+class PoiActivityTest {
+
+    @get: Rule
+    var testRule = ActivityScenarioRule(PoiActivity::class.java)
+
+    @Test
+    fun poiActivityTest() {
+
```

Comment:
Adding a  //todo here and explaining what needs to be done can be beneficial since someone can recognize he did something similar before and help. Also, Android Studio has a tool that shows you all the marked TODOs and it makes it easier to manage the work.